content/material/introduction.adoc

```diff
@@ -10,7 +10,7 @@ endif::[]
 == Motivation
 
 In virtual environments and simulations, the accurate representation of materials plays a critical role in achieving realism and ensuring proper interaction between objects and perception sensors.
-_Material properties_, such as roughness, permittivity, and index of refraction, influence how sensors like cameras, radars, and lidars perceive and interact with objects.
+_Material properties_, such as roughness, permittivity, and index of refraction, influence how sensors like cameras, radars, and LiDARs perceive and interact with objects.
```

Comment:
If lidar is spelled LiDAR, radar should be spelled RaDAR. However, I would prefer the spelling radar and lidar.

Reply:
Agreed, text is updated.